<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QtCharts::QChartView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>681</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QtCharts::QChartView" name="graphicsView_2">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>400</y>
      <width>671</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>681</width>
      <height>381</height>
     </rect>
    </property>
    <property name="title">
     <string>Edycja parametrów transmitancji</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>A2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>20</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>A1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>20</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>A0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>150</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Amplituda</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>150</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Okres wejścia</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>150</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Opóźnienie</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>250</y>
       <width>551</width>
       <height>121</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_A2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_A1">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>40</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_A0">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>40</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Amp">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>180</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Tin">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>180</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Opoz">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>180</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>20</y>
      <width>671</width>
      <height>371</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="p_char_faz">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>270</y>
       <width>101</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>
Charakterysytka
Fazowa
</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>401</width>
       <height>351</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="p_char_amp">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>140</y>
       <width>101</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>Charakterystyka
 Amplitudowa</string>
     </property>
    </widget>
    <widget class="QPushButton" name="p_syg_wyj">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>10</y>
       <width>101</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>Sygnał Wyjściowy</string>
     </property>
    </widget>
    <widget class="QPushButton" name="p_skok_jedn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>101</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>Skok Jednostkowy</string>
     </property>
    </widget>
    <widget class="QPushButton" name="p_fala_prost">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>101</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>Fala Prostokątna</string>
     </property>
    </widget>
    <widget class="QPushButton" name="p_sinus">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>101</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>Fala Sinusoidalna</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1400</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtCharts::QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QtCharts</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
